Week-9/todolist/src/App.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import './App.css';
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function Todo({ todo, index, markTodo, removeTodo }) {
+  return (
+    <div className="todo">
+      <span style={{ textDecoration: todo.isDone ? "line-through" : "" }}>
+        {todo.text}
+      </span>
+      <div>
+        <button
+          className="button green"
+          type="button"
+          onClick={() => markTodo(index)}
+        >
+          Done
+        </button>{" "}
+        <button
+          className="button red"
+          type="button"
+          onClick={() => removeTodo(index)}
+        >
+          Delete
+        </button>
+      </div>
+    </div>
+  );
+}
+
+function FormTodo({ addTodo }) {
+  const [value, setValue] = React.useState("");
+
+  const handleSubmit = (ele) => {
+    ele.preventDefault();
+    if (!value) return;
+    addTodo(value);
+    setValue("");
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="form-group">
+        <div>
+          <b>Add Todo</b>
+        </div>
+        <input
+          type="text"
+          className="form-control"
+          value={value}
+          onChange={(e) => setValue(e.target.value)}
+          placeholder="Add new todo"
+        />
+      </div>
+      <button className="button blue" type="submit">
+        Submit
+      </button>
+    </form>
+  );
+}
+
+function App() {
+  const [todos, setTodos] = React.useState([
```

Comment:
`import React, { useState } `

The above is a better way of including hooks than `React.useState`